crashlogger.js

```diff
@@ -8,16 +8,29 @@
  * @license MIT license
  */
 
+/// <reference types="nodemailer" />
```

Comment:
In other words, it is necessary to make the module transpile with no errors, but it depends on whether or not it'd be a good time to add `@types/nodemailer` as a dev dependency